.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@
 </details>
 
 <details>
-<summary>Associated [documentation repository](https://github.com/MarkUsProject/Wiki) pull request (if applicable)</summary>
+<summary>Associated <a href="https://github.com/MarkUsProject/Wiki"> documentation repository </a> pull request (if applicable)</summary>
```

Comment:
there shouldn't be any spaces between the anchor tags and the enclosing text (i.e., after the `<a>` and before the `</a>`)